src/components/operations/BuyCoinsOperation.tsx

```diff
@@ -0,0 +1,177 @@
+        <View>
+          <AssetImage nameImage={item.image.split('.')[0]} />
+          <Text style={styles.marginHorizontal}>
+            {translate(`wallet.operations.buy_coins.${item.description}`)}
+          </Text>
+          <Separator height={5} />
+          <View style={styles.centeredView}>
+            <ActiveOperationButton
+              title={translate('wallet.operations.buy_coins.title_buy')}
+              isLoading={false}
+              onPress={handleOnClick}
+              style={styles.buyButton}
+            />
+          </View>
+          <Separator height={15} drawLine={true} />
+        </View>
+      );
+    } else if (!item.name && item.image) {
+      return (
+        <View>
+          <AssetImage
+            onClick={handleOnClick}
+            nameImage={item.image.split('.')[0]}
+          />
+          <Separator height={10} />
+        </View>
+      );
+    }
+  };
+
+  const scrollToTop = () => {
+    if (flatListRef && flatListRef.current) {
+      (flatListRef.current as FlatList).scrollToIndex({
+        animated: true,
+        index: 0,
+      });
+    }
+  };
+
+  const {height} = useWindowDimensions();
+
+  const styles = getDimensionedStyles(height);
+
+  return (
+    <Operation
+      logo={<ShoppingCartIconBlack />}
+      title={translate('wallet.operations.buy_coins.title_buy')}>
+      <>
+        <Separator />
+        <View>
+          <View style={styles.rowContainer}>
+            <Text style={styles.switchText}>
+              {translate('wallet.operations.buy_coins.title_hive')}
+            </Text>
+            <Switch
+              trackColor={{false: '#767577', true: '#81b0ff'}}
+              thumbColor={isEnabled ? '#f5dd4b' : '#f4f3f4'}
+              ios_backgroundColor="#3e3e3e"
+              onValueChange={(value: boolean) => {
+                setIsEnabled(value);
+                scrollToTop();
+              }}
+              value={isEnabled}
+            />
+            <Text style={styles.switchText}>
+              {translate('wallet.operations.buy_coins.title_hbd')}
+            </Text>
+          </View>
+
+          {/* Flatlist one list */}
+          <FlatList
+            ref={flatListRef}
+            data={join2Arrays(
+              BuyCoinsListItem(
+                isEnabled ? BuyCoinType.BUY_HIVE : BuyCoinType.BUY_HDB,
+                user.name,
+              ).list,
+              BuyCoinsListItem(
+                isEnabled ? BuyCoinType.BUY_HIVE : BuyCoinType.BUY_HDB,
+                user.name,
+              ).exchanges,
+            )}
+            renderItem={(item) => renderItems(item.item)}
+            style={{maxHeight: '90%'}}
+          />
+        </View>
+      </>
+    </Operation>
+  );
+};
+const connector = connect((state: RootState) => {
+  return {
+    user: state.activeAccount,
+  };
+}, {});
+type PropsFromRedux = ConnectedProps<typeof connector>;
+
+const getDimensionedStyles = (width: number) =>
+  StyleSheet.create({
```

Comment:
Remove black border around exchanges logo 
+ check the exchanges still work with HIVE or HBD
+ change the logo when appropriate (the ones with white text don't render well)

Reply:
All checked, still working. SVGs updated.